pkg/webhook/nodenetworkconfigurationpolicy/conditions_test.go

```diff
@@ -0,0 +1,127 @@
+package nodenetworkconfigurationpolicy
+
+import (
+	"context"
+	"encoding/json"
+	"strconv"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+)
+
+var _ = Describe("Mutating Admission Webhook", func() {
+	type WebhookCase struct {
```

Comment:
What do you mean ?

Reply:
If there is only a single attribute here, maybe we could change the signature to:

`DescibeTable("when reset condition is called on a Policy", func(p ...Policy) {`

I won't fight for it. Just that it may be more clear on the first sight.